modules/olm-installing-from-operatorhub-using-cli.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * applications/operators/olm-adding-operators-to-cluster.adoc
+
+[id='olm-installing-operator-from-operatorhub-using-cli-{context}']
+= Installing from the OperatorHub using the CLI
+
+Instead of using the {product-title} web console, you can install an Operator
+from the OperatorHub using the CLI. Use the `oc` command as follows to install
+(subscribe to) an Operator for a selected namespace.
+
+.Prerequisites
+
+- Access to an {product-title} cluster using an account with `cluster-admin`
+permissions.
+
+.Procedure
+
+. View the list of Operators available to the cluster from the OperatorHub:
++
+----
+$ oc get packagemanifests -n openshift-marketplace
+----
+
+. To identify the Operators to install, create a CatalogSourceConfig object YAML
+file (for example, `csc.cr.yaml`) containing information similar to the
+following:
+
++
+.Example CatalogSourceConfig
+[source,yaml]
+----
+apiVersion: "marketplace.redhat.com/v1alpha1"
+kind: "CatalogSourceConfig"
+metadata:
+  name: "example"
+  namespace: "openshift-marketplace"
+spec:
+  targetNamespace: "global" <1>
+  packages: "myoperator,another-operator" <2>
+----
+<1> Set the `targetNamespace` to identify the namespace where you want the
+Operator to be available.
+<2> Set `packages` to a comma-separated list of Operators to which you want to
+subscribe.
+
+. Install the Operator to the selected namespace:
```

Comment:
Will the operator be deployed? There isn't a subscription object.

Reply:
@awgreene is correct. We need to fix our README too. We say "install" and it means something else. How about "CatalogSourceConfig is used to enable an operator present in the OperatorSource on your cluster."

Reply:
Would it be better to rename this procedure to "Enabling Operators from the OperatorHub using the CLI", or do we need to add more steps so that the full "Installing" procedure is accurate?

Reply:
I've added a Subscription creation step, per https://github.com/operator-framework/operator-marketplace/blob/master/README.md.

http://file.rdu.redhat.com/~adellape/022219/hub_cli_edits/applications/operators/olm-adding-operators-to-cluster.html#olm-installing-operator-from-operatorhub-using-cli-olm-adding-operators-to-a-cluster